scripts/build

```diff
@@ -11,8 +11,30 @@ if [ "$(uname)" = "Linux" ]; then
 fi
 LINKFLAGS="-X main.Version=$VERSION"
 LINKFLAGS="-X main.GitCommit=$COMMIT $LINKFLAGS"
-CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/backup-restore-operator
-if [ "$CROSS" = "true" ] && [ "$ARCH" = "amd64" ]; then
+if [ "$CROSS_ARCH" != "true" ]; then
+  CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/backup-restore-operator
+  if [ "$CROSS" = "true" ]; then
     GOOS=darwin go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-darwin
     GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/backup-restore-operator-windows
-fi
+  fi
+else
+  if [ "$ARCH" = "amd64" ]; then
```

Comment:
This could be a lot smaller of a change if we only want to support "cross arch" builds from ARM64->AMD64. However it felt like we should make it symmetrical to allow going the other direction. The primary use case for me needing this change to begin with could easily be inverted.

Especially with our continued investment into "edge" we may likely see more arm64 support needed overtime. And that will increase the odds of a developer compiling on `amd64` with their lab/target cluster being `arm64`.